exec/java-exec/src/main/java/org/apache/drill/exec/store/sys/store/DrillSysFilePathFilter.java

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
Please use comment for the header, not javadoc.

Reply:
OK. Will fix this.